usagov/vote-gov-drupal
# VOTE-2295: switch logo img to inline svg, add high contrast fill colors

## Jira ticket

[VOTE-2295](https://cm-jira.usa.gov/browse/VOTE-2295)

## Description

Changes logo fill colors on high contrast mode, replaces image tag with inline svg

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. npm run build
3. toggle high contrast mode on and off
4. confirm svg colors match

## Checklist for the Developer




## Checklist for the Peer Reviewers
